assets/server/static/js/popup/main.js

```diff
@@ -12,28 +12,35 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-window.addEventListener("DOMContentLoaded", async () => {
-  const originElement = document.getElementById("origin");
-  const windowElement = document.getElementById("windowname");
+const originElement = document.querySelector("#origin");
```

Comment:
I dont think the goal was to remove addEventListener, rather to just switch from getElementById to querySelector?

Reply:
Correct

Reply:
ack